api/src/maddo/infrastructure/repositories/replication-repository.js

```diff
@@ -94,6 +94,35 @@ export const replications = [
       return datamartKnex('target_profiles_course_duration').insert(chunk);
     },
   },
+  {
+    name: 'data-calibration',
```

Comment:
```suggestion
    name: 'data-calibrations',
```
Pour rester iso avec le nom des tables

Reply:
Merci !